Sources/Widgets/Widgets3D/ResliceCursorWidget/helpers.js

```diff
@@ -81,6 +94,35 @@ export function getViewPlaneNameFromViewType(viewType) {
   }
 }
 
+// Get the names and normals of other views
+// currentNormal = normal of the current view
+// It will return an object that contains
+export function getOtherViews(viewType, widgetState) {
+  const XPlaneNormal = widgetState.getXPlaneNormal();
+  const YPlaneNormal = widgetState.getYPlaneNormal();
+  const ZPlaneNormal = widgetState.getZPlaneNormal();
+
+  const viewX = {
+    name: 'X',
+    normal: XPlaneNormal,
+  };
+  const viewY = {
+    name: 'Y',
+    normal: YPlaneNormal,
+  };
+  const viewZ = {
+    name: 'Z',
+    normal: ZPlaneNormal,
+  };
+  let otherViews = [viewX, viewY];
+  if (viewType === ViewTypes.SAGITTAL) {
+    otherViews = [viewY, viewZ];
+  } else if (viewType === ViewTypes.CORONAL) {
+    otherViews = [viewX, viewZ];
+  }
+  return otherViews;
```

Comment:
I would split that function into 2 functions (1 function to get the other view types and 1 function to get the normal of a viewType).

But it seems to no longer be needed...

Reply:
Indeed.